docs/about/themes-translation.md

```diff
@@ -11,75 +11,39 @@ files generation and compilation. We worked on wrapping its usage through
 `setup.py` commands as described below to make updating and contributing
 translations easy.
 
-Make sure they are installed in your environment:
+Make sure translation requirements are installed in your environment:
 
 ```bash
 pip install -r requirements/project.txt
```

Comment:
If you do not have a strong opinion on this (you know that I don't either) I think we should stick with Jinja's extension naming which is `i18n`. This will be more intuitive to people.

Reply:
I think that makes sense. Installing `mkdocs[i18n]` enables Jinja's `i18n` extension.